File: exercism/csharp/raindrops/Raindrops.cs
using System;

public static class Raindrops
{
    public static string Convert(int number)
    {
        var rainspeak = "";
        rainspeak += ShouldPling(number) ? "Pling" : "";
        rainspeak += ShouldPlang(number) ? "Plang" : "";
        rainspeak += ShouldPlong(number) ? "Plong" : "";

        return rainspeak == "" ? $"{number}" : rainspeak;
    }

    private static bool ShouldPling(int number) => number % 3 == 0;
    private static bool ShouldPlang(int number) => number % 5 == 0;
    private static bool ShouldPlong(int number) => number % 7 == 0;
}

File: exercism/php/raindrops/raindrops.php
<?php

class Raindrops {

    private $number;

    public function __construct($number)
    {
        $this->number = $number;
    }

    public function speak()
    {
        $say = $this->number % 3 == 0 ? 'Pling' : '';
        $say .= $this->number % 5 == 0 ? 'Plang' : '';
        $say .= $this->number % 7 == 0 ? 'Plong' : '';
        return (string) ($say ?: $this->number);
    }
}

function raindrops($number) {
    return (new Raindrops($number))->speak();
}
